src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -465,6 +467,36 @@ RequestContext<object> requestContext
 
             await requestContext.SendResult(result.First().ToString());
         }
+        private async Task HandleGetCommandRequest(
+            string param,
+            RequestContext<object> requestContext)
+        {
+            PSCommand psCommand = new PSCommand();
+            if(param != "") {
+                psCommand.AddCommand("Get-Command").AddArgument(param);
```

Comment:
I wonder if we should module prefix these commands e.g. `Microsoft.PowerShell.Core\Get-Command`.  We've already had to do this to fix bugs that folks submitted because somewhere they were using a command proxy that didn't work quite right.  :-(

Reply:
Consider it done.